netbench-orchestrator/src/ssm_utils/netbench_driver/s2n_quic_driver_crates.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{CrateIoSource, NetbenchDriverType};
+
+pub fn s2n_quic_server_driver() -> NetbenchDriverType {
```

Comment:
Could we just use the `github` source rather than having both the crates.io and the github source?

Reply:
We need to support both so that its possible to test the latest from github while also testing old crates.io versions